docs/AutomationReference.md

```diff
@@ -280,12 +303,25 @@ These versions embed the symbols into the assemblies and support [SourceLink](ht
 
 #### Error handling
 
-`AxeWindowsAutomationException` is thrown for all unhandled errors in Axe.Windows.Automation. If an exception was thrown from code not owned by Axe.Windows.Automation, that exception will be wrapped in the `Exception.InnerException` property.
+`AxeWindowsAutomationException` is thrown for errors in `Axe.Windows.Automation`'s logic where meaningful error reporting can be generated. In some situations, the `Exception.InnerException` property may contain a corresponding system-level exception for errors encountered by Axe.Windows. Other exceptions may be thrown by the system from calls to this library.
+
+### Migrating from Axe.Windows 1.x
+Migration from Axe.Windows 1.x to Axe.Windows 2.0 should require minimal code changes for most projects.
+
+#### `IScanner.ScanAll` removed
+The `ScanAll` method for scanning multiple top-level windows has been removed. Use `ScanAsync` or `Scan` instead.
 
-#### Fully synchronous
-Because  automated scans are stateful, they are intentionally synchronous within
-a process. If you attempt to initiate multiple scans concurrently, the first one
-to obtain the lock will execute, then another, then another. This is by design
-and is not expected to change at any future time. If you have a scenario that
-truly requires the command to execute in parallel, then you will need to create
-a solution where you can make those calls from separate processes.
+#### Calling `IScanner.Scan`
+
+In place of this | Do this
+---|---
+`IScanner.Scan()` | `IScanner.Scan(null)`
+`IScanner.Scan("scanId")` | `IScanner.Scan(new ScanOptions(scanId: "scanId"))`
+
+#### `ScanResults` replaced with `ScanOutput`
+The new return type of `IScanner`'s methods is `ScanOutput`. This object contains a `WindowScanOutputs` field, a `IReadOnlyCollection` of `WindowScanOutput` objects. These objects contain the same fields as the previous `ScanResults` class.
```

Comment:
This would be a prime example of internal reference, something like this:
```suggestion
The new return type of `IScanner`'s methods is [`ScanOutput`](Link to ScanIOutput in doc
```suggestion
`Scan` returns a [`ScanOutput`](https://github.com/microsoft/axe-windows/blob/main/docs/AutomationReference.md#scanoutput) 
```

The long-term goal is to have things described in detail in exactly one place to avoid the risk of things drifting out of sync.

Reply:
I'm worried about linking to one particular branch. What if someone forks?